docs/src/reference/callbacks.md

```diff
@@ -0,0 +1,6 @@
+# Callback API
+
```

Comment:
Maybe have a link to the manual section from each reference section in case someone arrives at this page from Google or whatever.